lib/opset.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import {Graph} from './graph';
+import {Operator} from './operators';
+
+export interface OpSet {
+  domain: string;
+  version: number;
+}
+
+export declare namespace OpSet {
+  interface OperatorConstructor {
+    (node: Graph.Node): Operator;
+  }
+
+  /**
+   * Domain of an opset, it can be an empty string(default value, represent for ai.onnx), or 'ai.onnx.ml'
```

Comment:
can the domain be ai.onnx OR default value (empty string) to represent the ai.onnx domain ?

Reply:
empty string means 'ai.onnx' as in domain